benefits/static/css/admin/styles.css

```diff
@@ -193,6 +193,62 @@ iframe.card-collection {
   display: none;
 }
 
+/* Login Page link styles */
+.login a:not(.btn):not(.card) {
+  color: var(--primary-color);
+  text-decoration: underline;
+  text-decoration-style: solid;
+  font-weight: var(--bold-font-weight);
+  border-radius: var(--focus-border-radius);
+}
+
+.login a:hover:not(.btn) {
+  color: var(--hover-color);
+  text-decoration: underline;
+}
+
+.login a:focus:not(.btn):not(.card):not(.footer-link):not(#skip-to-content),
+.login
+  a:focus-visible:not(.btn):not(.card):not(.footer-link):not(#skip-to-content) {
+  outline: var(--focus-style) !important;
+  outline-offset: var(--focus-outline-offset) !important;
+}
+
+.login a:visited:not(.btn) {
+  color: var(--selected-color);
+}
+
+:root {
+  --external-link-icon-size: var(--font-size-24px);
+  --external-link-icon-size-small: var(--font-size-18px);
+}
+
+.login a[target="_blank"]::after {
+  content: " ";
+  background-color: currentColor;
+
+  mask-image:
+    url("/static/img/external-link.svg"),
+    linear-gradient(transparent, transparent);
+  mask-position: center center;
+  -webkit-mask-image:
+    url("/static/img/external-link.svg"),
+    linear-gradient(transparent, transparent);
+  -webkit-mask-position: center center;
+
+  display: inline-block;
+  position: relative;
+  width: var(--external-link-icon-size);
+  height: var(--external-link-icon-size);
+  margin-left: 2px;
+  vertical-align: text-bottom;
+}
+
+/* reCAPTCHA: Do not display badge */
+.grecaptcha-badge {
+  display: none;
+}
+
```

Comment:
This is how I would recommend fixing the link color issue.

The link is getting the white color from this variable, `--link-fg`. Our `styles.css` file is setting `--link-fg` to white for the whole app (`:root`). We can set the variable to a separate value _just_ for the `login` page with:


![image](https://github.com/user-attachments/assets/5071a9c9-6ea1-4939-ae1d-69bb1c3ca8da)

Reply:
This doesn't address all the other styles though, right? Like the hover, focus rings, external link icon... we still need that right?

Reply:
Yeah we still need it. This is just merely adding those 3 lines to address the white color:

```.login {
   --link-fg: var(--bs-body-color-rgb);
 }
 ```

Reply:
I have not done a full full review of this CSS, just trying to get the button and link in a normal state. I suspect the button CSS code can be substantially improved but that would require a lot more thought/time.